ibm/mas_devops/roles/db2/tasks/install/main.yml

```diff
@@ -157,11 +157,19 @@
 - name: "Delete old db2 subscription, operand request and csv from {{ ibm_common_services_namespace }}"
   include_tasks: "tasks/delete_db2_operand_request.yml"
 
-- name: "Create Db2 Universal Operator Subscription in {{ db2_namespace }} namespace"
+- name: "Create db2u namespace operatorgroup"
```

Comment:
The operatorgroup will be created by `common_tasks/create_subscription.yml` (if one is needed) so we can also remove this (and the associated [templates/db2u_operatorgroup.yml.j2`) ...

Reply:
Thanks for the feedback.  Removed the call and the template.

Reply:
Updated in the last commit.